api/routes/user.py

```diff
@@ -112,7 +116,7 @@ def update_user(
             )
 
         req_user = UserUpdate(**req_user.__dict__)
-    elif not req_user.is_admin:
+    elif not req_user.is_admin and isinstance(req_user.is_admin, bool):
```

Comment:
Why do you need this type check?

Reply:
I used it here because the `is_admin` is an `Optional` field and as default, it will be a `None`. I expected that the admin can not downgrade admins to common users.